### PR TITLE
created readme

### PR DESCRIPTION
**Pull request by marchenko-vitalii**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 target/*
-*.iml
\ No newline at end of file
+*.iml
+*logs
```
Comment (SergiiSV) [inline .gitignore]: does "logs" should be like "logs/*" like your folders?

File: src/main/resources/log4j2.xml  [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <Loggers>
+        <Logger name="mate" level="info" additivity="false">
+            <AppenderRef ref="LogToConsole"/>
+            <AppenderRef ref="LogToFile"/>
+        </Logger>
+        <Root level="error">
+            <AppenderRef ref="Console"/>
+        </Root>
+    </Loggers>
+</Configuration>
```
Comment (SergiiSV) [inline src/main/resources/log4j2.xml ]: add empty line

**Review by SergiiSV — CHANGES_REQUESTED** [review]
2 coments

**Review by SergiiSV — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
 # taxi-service
```
Comment (Nick97-git) [inline README.md]:
```suggestion
# Taxi-service
```

File: README.md [hunk 4]
```diff
@@ -1 +1,40 @@
...
+
+This is simple program for taxi service. Program has next functions:
+- track all available cars and working drivers
+- register new drivers
+- add new cars and their manufacturers
+- delete cars, drivers and manufacturers
+- add drivers to cars
+- track your cars as a driver
```
Comment (Nick97-git) [inline README.md]: rewrite this part, it's very similar to one of the examples

File: README.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+- Servlet
+- JSTL
+- JSP
+- HTML, CSS
+---
+## Data Base Diagram of project
+
+![][logo]
+
+[logo]: https://mate-academy-images.s3.eu-central-1.amazonaws.com/car_diagram_db_2_4f50942103.png
```
Comment (Nick97-git) [inline README.md]: remove this image or create your own, it's very popular

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 6]
```diff
@@ -14,14 +14,14 @@
...
-    <tr><td><a href="${pageContext.request.contextPath}/manufacturers/">Display All Manufacturers</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/drivers">Display All Drivers</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/cars">Display All Cars</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/manufacturers">Display All Manufacturers</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/drivers/add">Create new Driver</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/cars/add">Create new Car</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/manufacturers/add">Create new Manufacturer</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/cars/drivers/add">Add Driver to Car</a></td></tr>
-    <tr><td><a href="${pageContext.request.contextPath}/cars/bydriver">All my Cars</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/cars/by_driver">All my Cars</a></td></tr>
```
Comment (Nick97-git) [inline src/main/webapp/WEB-INF/views/index.jsp]:
```suggestion
    <tr><td><a href="${pageContext.request.contextPath}/drivers/cars">All my Cars</a></td></tr>
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**